### ISSUE TITLE
Elshimo Marble

### ISSUE THREAD

**Issue by merkaga** [HomepointXI/Issues#275]
**Date & Time**: 
10/28/2018

**Client Version (use `/ver` in game):**

**Character Name:**

Merkaga

**Discord Name (if different from in game):**

**Nation:**
Windurst

**Job(level)/Sub Job(level):**

64 BLM/Whm

**NPC or Monster or item Name:**

Ivory Lizard/Elshimo Marble

**Zone name:** 

**Coordinates (use `!where`):**

Missionary man Quest

**Item tool link (for items issues only):**

**Multi-boxing? (multiple clients on same connection):**

**Steps to reproduce:**

**Any additional information:**

**Comment by Heepster03** [comment]
Yeah... this ticket is missing some vital info. No additional info is filled in, don't know the issue besides it being the Marble.

**Comment by nfaletra** [comment]
You’re a marble

**Comment by Heepster03** [comment]
Drop is listed as 2% drop rate from the lizards. Bring some TH with you.

**Comment by Heepster03** [comment]
I'm a red marble, don't think that's what they are looking for.

**Comment by Heepster03** [comment]
@merkaga, you do know we have custom tele commands right? Don't need the spells unless you really want it.

**Comment by EpicTaru** [comment]
http://wiki.homepointxi.com/Player_Commands

**Comment by merkaga** [comment]
Oh Wow...Thank You =)



-----Original Message-----

**Comment by nfaletra** [comment]
Wait a full day and zone before talking to Mulaujeant again. The quest is still following the old method. Retail updated it to be faster. DSP has not caught up yet.

**Comment by merkaga** [comment]
I talked to mulaujeant and nothing with the quest happens,i tried getting marble to drop with treasure hunter 1 and no luck. Thank you Roy

**Comment by Jessland86** [comment]
![image](https://user-images.githubusercontent.com/28382433/47891656-8e2b5f80-de2a-11e8-8d5c-0ccbebf9343b.png)

Dropped second kill. the marble does drop regardless if you are on the quest or not.

**Comment by Jessland86** [comment]
If TH1 is all you have you might want to ask around for some help from someone with higher TH.
